go/test/endtoend/cluster/vtctld_process.go

```diff
@@ -66,6 +66,14 @@ func (vtctld *VtctldProcess) Setup(cell string, extraArgs ...string) (err error)
 		"--port", fmt.Sprintf("%d", vtctld.Port),
 		"--grpc_port", fmt.Sprintf("%d", vtctld.GrpcPort),
 	)
+
+	if v, err := GetMajorVersion("vtctld"); err != nil {
+		return err
+	} else if v >= 18 {
+		vtctld.proc.Args = append(vtctld.proc.Args, "--bind_address", "127.0.0.1")
+		vtctld.proc.Args = append(vtctld.proc.Args, "--grpc_bind_address", "127.0.0.1")
```

Comment:
New flags should use dash and not underscore as a separator.

Reply:
Fixed via 4fd296bc2b